packages/natds-themes/build/html/avon/dark.html

```diff
@@ -14,6 +14,10 @@
       border: 1px solid #eaeaea;
       height: 50px;
     }
+
+    .size-sample {
```

Comment:
Curiosidade: o que seria esse `.size-sample`?

Reply:
é uma classe com as definições especificas para os blocos de tokens `size` para o sample html (to seguindo o padrão color-sample, size-sample) pois são blocos com "samples" dos tokens. Faz sentido?

Reply:
Faz sim. Obrigado :D